ui/helpers/constants.js

```diff
@@ -213,6 +213,7 @@ export const thEvents = {
   openLogviewer: 'open-logviewer-EVT',
   autoclassifyIgnore: 'ac-ignore-EVT',
   applyNewJobs: 'apply-new-jobs-EVT',
+  filtersUpdated: 'filters-updated-EVT',
```

Comment:
I needed this event due to removing the app reload when a range of pushes is changed.  In some cases, we want to set the ``fromchange`` value, but not have that trigger a ``hashchange`` event that would then reload the list of pushes.